contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -144,7 +153,7 @@ contract VaultAdmin is VaultStorage {
      */
 
     function removeStrategy(address _addr) external onlyGovernor {
```

Comment:
Can we have a similar function that takes index as an argument instead of address? Could be helpful in saving gas sometimes.